src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/adders/ObtainedDateTooltip.java

```diff
@@ -70,4 +70,22 @@ public static String getTimestamp(ItemStack stack) {
 
 		return "";
 	}
+
+	/**
+	 * @see #getTimestamp(ItemStack)
+	 */
+	public static long getLongTimestamp(ItemStack stack) {
```

Comment:
Should all timestamps be long?

Reply:
I could make the other method call this, I wasn't particularly happy about the code duplication either

Reply:
Fixed